handler.js

```diff
@@ -35,6 +36,7 @@ module.exports.corsProxy = (event, context, callback) => {
             method: event.httpMethod,
             timeout: 20000,
```

Comment:
I would also suggest reducing the timeout to something more sensible. I don't know if the author still supports this, but I've just needed it and kinda fought with the same issues that this PR solves.